plutus-core/plutus-ir/test/PlutusIR/Scoping/Tests.hs

```diff
@@ -23,54 +22,54 @@ import PlutusIR.Transform.Unwrap
-    , test_scopingGood "'commuteDefaultFun'" genTerm BindingRemovalNotOk PrerenameYes $
+    , T.test_scopingGood "'commuteDefaultFun'" genTerm T.BindingRemovalNotOk T.PrerenameYes $
         pure . commuteDefaultFun
     , -- We say that it's fine to remove bindings, because they never actually get removed,
       -- because the scope checking machinery doesn't create unused bindings, every binding
       -- gets referenced at some point at least once (usually very close to the binding site).
       -- So this test doesn't really test much.
-      test_scopingGood "dead code elimination" genTerm BindingRemovalNotOk PrerenameNo $
+      T.test_scopingGood "dead code elimination" genTerm T.BindingRemovalNotOk T.PrerenameNo $
         removeDeadBindings def
-    , test_scopingGood "constant folding" genTerm BindingRemovalNotOk PrerenameYes $
+    , T.test_scopingGood "constant folding" genTerm T.BindingRemovalNotOk T.PrerenameYes $
         pure . evaluateBuiltins False def defaultBuiltinCostModel
     , -- At the moment the inliner does not preserve global uniqueness (contrary to what it
       -- promises) due to the lack of marking in it (or initial renaming of the entire program,
       -- which would perform marking too).
-      test_scopingBad "inlining" genTerm BindingRemovalOk PrerenameYes $
+      T.test_scopingBad "inlining" genTerm T.BindingRemovalOk T.PrerenameYes $
         Inline.inline mempty def
-    , test_scopingGood "match-against-known-constructor" genTerm BindingRemovalNotOk PrerenameNo $
+    , T.test_scopingGood "match-against-known-constructor" genTerm T.BindingRemovalNotOk T.PrerenameNo $
         knownCon
-    , test_scopingGood "floating bindings inwards" genTerm BindingRemovalNotOk PrerenameNo $
+    , T.test_scopingGood "floating bindings inwards" genTerm T.BindingRemovalNotOk T.PrerenameNo $
         In.floatTerm def True
     -- Can't test 'Out.floatTerm', because it requires the type of annotations to implement
     -- 'Semigroup' and it's not clear what that means for 'NameAnn'.
-    , test_scopingGood "merging lets" genTerm BindingRemovalNotOk PrerenameYes $
+    , T.test_scopingGood "merging lets" genTerm T.BindingRemovalNotOk T.PrerenameYes $
         pure . letMerge
     , -- The pass breaks global uniqueness, but it's not clear whether this is by design or not.
-      test_scopingBad "compilation of non-strict bindings" genTerm BindingRemovalOk PrerenameYes $
+      T.test_scopingBad "compilation of non-strict bindings" genTerm T.BindingRemovalOk T.PrerenameYes $
         compileNonStrictBindings True
-    , test_scopingGood "match-against-known-constructor" genTerm BindingRemovalNotOk PrerenameYes $
+    , T.test_scopingGood "match-against-known-constructor" genTerm T.BindingRemovalNotOk T.PrerenameYes $
         pure . recSplit
-    , test_scopingGood "renaming" genProgram BindingRemovalNotOk PrerenameNo $
+    , T.test_scopingGood "renaming" genProgram T.BindingRemovalNotOk T.PrerenameNo $
```

Comment:
redundant `$`

Reply:
It's there for consistency, I think I'm fine with it.